assets/training/model_management/environments/text-generation-inference/context/requirements.txt

```diff
@@ -0,0 +1,18 @@
+git+https://github.com/huggingface/transformers.git@main
+text-generation==0.6.0
+pandas==2.0.3
+numpy==1.25.0
+azureml-inference-server-http==0.8.4
+azureml-core==1.52.0
+azureml-contrib-services==1.52.0
+psutil==5.9.5
+mlflow==2.3.1
+azure-ai-contentsafety==1.0.0b1
+aiolimiter==1.1.0
+azureml-ai-monitoring==0.1.0b3
+azure-mgmt-cognitiveservices==13.4.0
+azure-identity==1.13.0
+cryptography==41.0.3
```

Comment:
you should not have cryptography, certifi, requests specified

Reply:
removed

Reply:
removed

Reply:
removed

Reply:
removed